src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -382,8 +386,11 @@ private void reconfigureActions() {
 		configureFileUpload();
 		configureProvenance();
 		configureChangeStorageLocation();
-		configureCreateDOI();
-		configureCreateOrUpdateDoi();
+		if (DisplayUtils.isInTestWebsite(cookies)) {

```

Comment:
this is not the right place for this check (since the command visibility is set in the configure methods).  Please unconditionally hide old command in alpha mode (in configure method), and unconditionally hide the new command in non-alpha mode (in new configure method).